app/serializers/activitypub/context_serializer.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class ActivityPub::ContextSerializer < ActivityPub::Serializer
+  include RoutingHelper
+
+  attributes :id, :type, :first, :attributed_to
+
+  has_one :first, serializer: ActivityPub::CollectionSerializer
+
+  def id
+    ActivityPub::TagManager.instance.uri_for(object)
+  end
+
+  def attributed_to
+    ActivityPub::TagManager.instance.uri_for(object.parent_account)
+  end
+
+  def type
+    'Collection'
+  end
+
+  def first
+    conversation_statuses = object.statuses[0..5]
+    last_status = conversation_statuses.last
+    conversation_statuses = conversation_statuses.pluck(:id, :uri)
+    last_id = conversation_statuses.last&.first
+    has_more = object.statuses.count > ActivityPub::ContextsController::DESCENDANTS_LIMIT
```

Comment:
A `COUNT()` is an expensive operation. What you could do is fetch `ActivityPub::ContextsController::DESCENDANTS_LIMIT + 1` items and then see if you've gotten more than `ActivityPub::ContextsController::DESCENDANTS_LIMIT` or not. Ideally this would be the top query.